cloudbuild.yaml

```diff
@@ -1,11 +1,6 @@
 steps:
-- name: 'gcr.io/cloud-builders/go'
-  args: ['install', '.']
-  env: ['PROJECT_ROOT=hello']
-- name: 'gcr.io/cloud-builders/go'
-  args: ['build', 'hello']
-  env: ['PROJECT_ROOT=hello']
-artifacts:
-  objects:
-    location: 'gs://$PROJECT_ID/'
-    paths: ['hello']
+- name: golang
+  args: ['go', 'mod', 'init', 'github.com/your/import/path']
+- name: golang
+  env: ['GO111MODULE=on']
```

Comment:
This env is set already inside the golang image, so this isn't necessary.

Reply:
removed.